Implemented Confirmation Message Popup of Link Copied

kesharibhai84 opened the pull request:
…his function displays a popup message when a link is copied. Updated createSocialLinks Function: This function now calls showCopyMessage to show the popup when a link is copied.

# Title
Fixes #56: Implement Confirmation Message Popup of Link Copied

## Description
Added showCopyMessage Function: This function displays a popup message when a link is copied.
Updated createSocialLinks Function: This function now calls showCopyMessage to show the popup when a link is copied.
added some css style too for the copy confirmation.
there was one more change i made in html file for showing popup text.
---
## Issue Ticket Number
Fixes #56 

---
## Type of change
<!-- Please select all options that are applicable. -->
- [X] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

---
# Checklist:
- [X] I have followed the contributing guidelines of this project as mentioned in [CONTRIBUTING](./docs/CONTRIBUTING)
- [X] I have checked to ensure there aren't other open [Pull Requests](https://github.com/Param302/SocialRepo/pulls) for the same update/change?
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes needed to the documentation

kesharibhai84 committed:
made some changes in html , css and Added showCopyMessage Function: This function displays a popup message when a link is copied. Updated createSocialLinks Function: This function now calls showCopyMessage to show the popup when a link is copied.

Param302 reviewed (changes requested):
Hey,

I can see you've removed a lot of existing JS code, please don't do this, you just have to write your code at where I specified, you are not asked to remove existing JS code.

Please fix it.

kesharibhai84 committed:
In html: there was little change to show the text for copy popup.
In css: Added Styles for the copy confirmation popup.
In index.js : Only modifed the showCopyMessage.

kesharibhai84 committed:
Fixed #56:made main changes in showCopyMessage

kesharibhai84 commented:
i have raised new pr check that one.
